python/whylogs/core/datatypes.py

```diff
@@ -107,6 +107,57 @@ def _do_match(cls, dtype_or_type: Any, maybe_type: Optional[Any]) -> bool:
         return False
 
 
+class Tensor(DataType[Any]):
+    def __init__(self) -> None:
+        super().__init__(str)
```

Comment:
`str`?

Reply:
Ah, this is just pasted from `String` and still needs work?

Reply:
Correct. Lots of work.